src/app/beer_garden/api/http/handlers/v1/job.py

```diff
@@ -57,6 +57,12 @@ async def patch(self, job_id):
           ```JSON
           { "operation": "update", "path": "/status", "value": "RUNNING" }
           ```
+
+          You can also execute an existing job with:
+          ```JSON
+          { "operation": "update", "path": "/execute", "value": "True" }
+          { "operation": "update", "path": "/reset_interval", "value": "true" } <-- how do this?
```

Comment:
The existing API is kind of weird and thus not something I would suggest emulating here. And even if we liked the existing pattern, I'm not sure it applies here.  The status update stuff at least is in line with what a PATCH operation implies (making a partial change to the resource), but an adhoc execution of a scheduled job isn't really changing said job, so PATCH doesn't feel like the appropriate method.  See the approach suggested in the original issue write-up.  If there were specific concerns or disagreements about that we can certainly discuss it.